src/composables/transactionFilter.ts

```diff
@@ -0,0 +1,39 @@
+import { computed, ref } from '@vue/composition-api';
+import { IFilterInputPayload, ObjectValues } from '../types';
+
+const FILTER_MODE = {
+  all: 'all',
+  in: 'in',
+  out: 'out',
+  dex: 'dex',
+} as const;
+
+type TransactionsFilterMode = ObjectValues<typeof FILTER_MODE>;
+
+type TransactionsFilterPayload = IFilterInputPayload<TransactionsFilterMode>;
+
+function createEmptyFilter(): TransactionsFilterPayload {
+  return { rotated: true, key: FILTER_MODE.all };
+}
+
+const searchPhrase = ref<string>('');
+const displayMode = ref<TransactionsFilterPayload>(createEmptyFilter());
+
+export const useTransactionFiler = () => {
+  const expandSearchBarAndFilter = computed(
```

Comment:
As this is a boolean I suggest to use name that suggest this. For example `isSearchBarAndFilterExpanded`. Now the name suggests that it's a method/function.